WcaOnRails/app/webpacker/components/CompetitionOverview/CompetitionFilters.js

```diff
@@ -0,0 +1,662 @@
+                competitionData={recentComps}
+                title={I18n.t('competitions.index.titles.recent', { count: competitionConstants.competitionRecentDays })}
+                showRegistrationStatus={showRegistration}
+                showCancelled={showCancelled}
+                selectedEvents={selectedEvents}
+                loading={competitionsIsFetching}
+                loaded={!hasUnloadedCompetitions}
+              />
+            )
+          }
+          {
+            displayMode === 'list'
+            && timeOrder === 'past'
+            && (
+              <CompetitionTable
+                competitionData={pastComps[pastSelectedYear]}
+                title={pastSelectedYear === 'all_years' ? I18n.t('competitions.index.titles.past_all') : I18n.t('competitions.index.titles.past', { year: pastSelectedYear })}
+                showRegistrationStatus={showRegistration}
+                showCancelled={showCancelled}
+                selectedEvents={selectedEvents}
+                loading={competitionsIsFetching}
+                loaded={!hasUnloadedCompetitions}
+              />
+            )
+          }
+          {
+            displayMode === 'list'
+            && timeOrder === 'by_announcement'
+            && (
+              <CompetitionTable
+                competitionData={sortByAnnouncementComps}
+                title={I18n.t('competitions.index.titles.by_announcement')}
+                showRegistrationStatus={showRegistration}
+                showCancelled={showCancelled}
+                selectedEvents={selectedEvents}
+                sortByAnnouncement
+                loading={competitionsIsFetching}
+                loaded={!hasUnloadedCompetitions}
+              />
+            )
+          }
+          {
+            displayMode === 'list'
+            && timeOrder === 'custom'
+            && (
+              <CompetitionTable
+                competitionData={customDatesComps}
+                title={I18n.t('competitions.index.titles.custom')}
+                showRegistrationStatus={showRegistration}
+                showCancelled={showCancelled}
+                selectedEvents={selectedEvents}
+                loading={competitionsIsFetching}
+                loaded={!hasUnloadedCompetitions}
+              />
+            )
+          }
+        </div>
+        {/* old code does a lot of things to #competitions-map... to be included? */}
+        <div name="competitions-map">
+          {
+            displayMode === 'map'
+            && (
+              <CompetitionMap
+                competitionData={mapDisplayComps}
+                selectedEvents={selectedEvents}
+                showCancelled={showCancelled}
+              />
+            )
+          }
+        </div>
+      </Container>
+
+      {!competitionsIsFetching && hasUnloadedCompetitions && displayMode === 'list' && <div ref={ref} name="page-bottom" />}
+    </Container>
+  );
+}
+
+export default CompetitionFilter;
```

Comment:
Default export should match the file name (file is plural, function is singular).